src/pymedphys/mephysto/_level1/mccread.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (C) 2017 Cancer Care Associates
```

Comment:
What does "mc" refer to in "mcread.py"

Reply:
The file extension of PTW Mephysto files `.mcc`